web/client/actions/security.js

```diff
@@ -18,6 +19,12 @@ const LOGOUT = 'LOGOUT';
 
 
 function loginSuccess(userDetails, username, password, authProvider) {
+    // letting security utils know about a successfull login
+    SecurityUtils.provideSecurityInfo({
+        user: userDetails && userDetails.User,
+        username: username,
```

Comment:
this should stay in the reducer, and the password should not be saved (basic header instead), follow how we do it with GeoStore utilities.